applications/napari-ls6/README.md

```diff
@@ -0,0 +1,25 @@
+# About
+Napari is a fast, interactive, multi-dimensional image viewer for Python, designed for browsing, annotating, and analyzing large 2D, 3D, and higher-dimensional datasets
+
+## App build details
+uses Apptainer
```

Comment:
Also uses DCV, taccaci/interactive-base:1.1.0, and runs on ls6 as opposed to a VM (a lot of our interactive apps run on VMs).

Probably outside the scope of this to actually resolve the issue, but worth calling out that the container seems to pull from Erik's personal Dockerhub - `eriksf/napari:latest`